### ISSUE TITLE
Some sound events are not working properly (in game) while other event are fine.

### ISSUE THREAD

**Issue by NukprachTH** [GaijinEntertainment/fmod_studio_warthunder_for_modders#234]
So now, I have problem with shell sound that should sound like this (cannon_shell_drop)

https://github.com/GaijinEntertainment/fmod_studio_warthunder_for_modders/assets/111956181/e57de7bc-9c64-45c1-8a42-7a080b70f259

but when in game, it play default sound.

https://github.com/GaijinEntertainment/fmod_studio_warthunder_for_modders/assets/111956181/0a0ddc17-d939-4e6d-a167-65aff915c397

I've tried all user manual about wt sound modding but it still doesn't help.

**I'm already using Fmod version 2.01.06 for wt sound modding**

_-have this same problem with all MGs too_

It's weird that it only happen to some sounds, so here's one that work properly (cannon_shell_med_drop)

https://github.com/GaijinEntertainment/fmod_studio_warthunder_for_modders/assets/111956181/930b19ae-36d9-4855-b295-9499998a78ac

https://github.com/GaijinEntertainment/fmod_studio_warthunder_for_modders/assets/111956181/956401f0-1e95-493e-8495-832e5c0a09dc

**Comment by Uchuujin01** [comment]
Hello! Are you sure you work with that cannon sound event that sounds in tank you test it in?

**Comment by NukprachTH** [comment]
Yes, I'm very sure that I'm working on cannon sound event that sounds in tank I'm test it in.
